charts/sscs-bulk-scan/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 name: sscs-bulk-scan
 home: https://github.com/hmcts/sscs-bulk-scan
-version: 0.0.3
+version: 0.0.4
```

Comment:
Can you update your chart-java version to 2.9.1 please:
https://github.com/hmcts/sscs-bulk-scan/blob/master/charts/sscs-bulk-scan/requirements.yaml#L3